apps/extension/src/App/Accounts/AccountListing.tsx

```diff
@@ -6,43 +7,59 @@ import {
   Details,
   Alias,
   DerivationPath,
-  CopyToClipboard,
+  Button,
 } from "./AccountListing.components";
 
-import { DerivedAccount } from "background/keyring";
+import { DerivedAccount, KeyStoreType } from "background/keyring";
+import { TopLevelRoute } from "App/types";
 
 type Props = {
+  // The parent Bip44 "account"
+  parent?: number;
+  // The child account
   account: DerivedAccount;
+  parentAlias: string;
 };
 
 const textToClipboard = (content: string): void => {
   navigator.clipboard.writeText(content);
 };
 
-const AccountListing = ({ account }: Props): JSX.Element => {
-  const { address, alias, bip44Path, establishedAddress } = account;
-  const coinType = "0'";
+const AccountListing = ({ account, parentAlias }: Props): JSX.Element => {
+  const { address, alias, path, establishedAddress, type } = account;
+  const navigate = useNavigate();
 
   return (
     <AccountListingContainer>
       <Details>
-        {alias && <Alias>{alias}</Alias>}
         <DerivationPath>
-          m/44'/{coinType}/{`${bip44Path.account}'`}/{bip44Path.change}/
-          {bip44Path.index}
+          {type !== KeyStoreType.Mnemonic &&
+            `${parentAlias} /${path.account}'/${path.change}${
+              typeof path.index === "number" ? "/" + path.index : ""
+            }`}
         </DerivationPath>
```

Comment:
nit: I feel that moving logic out of "html" adds readability and makes it less error-prone
```suggestion
  const navigate = useNavigate();
  
  // if we have a non-mnemonic account, we display the path
  const isNonMnemonicAccountType = type !== KeyStoreType.Mnemonic;
  const maybePathIndex = typeof path.index === "number" ? "/" + path.index : "";
  const formattedDerivationPath = isNonMnemonicAccountType
    ? `${parentAlias} /${path.account}'/${path.change}${maybePathIndex}`
    : "";
  return (
    <AccountListingContainer>
      <Details>
        <DerivationPath>
          {formattedDerivationPath}
        </DerivationPath>
```